_source/_posts/2023-07-27-spring-graphql-react.adoc

```diff
@@ -0,0 +1,1857 @@
+---
```

Comment:
I love that you used AsciiDoc! Unfortunately, the Auth0 blog only supports Markdown. I can handle the translation for you if you don't have time.

Reply:
I migrated to Markdown